misc/python/materialize/github.py

```diff
@@ -63,12 +65,13 @@ def get_known_issues_from_github_page(token: str | None, page: int = 1) -> Any:
 
 def get_known_issues_from_github(
     token: str | None = os.getenv("GITHUB_TOKEN"),
+    repo: str = "MaterializeInc/database-issues",
```

Comment:
We don't set this parameter anywhere. Shouldn't we loop over all repos to include, for example, the cloud repo in the closed-issues detector as well?